### DISCUSSION TITLE
/software-systems-engineering/sql-server-and-uuids/

### DISCUSSION THREAD

**Discussion by mlhaufe** [thenewobjective/thenewobjective.github.io#87] [Announcements]
Comment thread for https://thenewobjective.com/software-systems-engineering/sql-server-and-uuids

**Comment by gildas-lormeau** [comment]
There's a typo in the section ["Choosing a key format"](https://thenewobjective.com/software-systems-engineering/sql-server-and-uuids#choosing-a-key-format), the "n" is missing in the word "can", cf. "[...] you *can* have collisions if they are using the same scheme."

Also, it might be useful to set `word-break` to `initial` in the rule `.post > table th` to avoid breaking words in the table headers. For example the "Version" label is broken in two. I really like the design of your site though!

**Comment by mlhaufe** [comment]
Adding `text-align: left` for tables seems to be a big improvement as well. This didn't show up as a problem on my tablet where I authored this. 

For the typo:  I wish there were better tools for markdown here. It didn't show up as a problem in VSCode. I'll hunt around for a better extension.

I appreciate the feedback.

**Comment by mlhaufe** [comment]
Here is an npm module referencing this post:

https://www.npmjs.com/package/@vulcan.sh/id

**Comment by mlhaufe** [comment]
This article has been reposted on LinkedIn: https://www.linkedin.com/pulse/sql-server-uuids-michael-haufe-g4v0c

**Comment by mlhaufe** [comment]
A quick gist:

https://gist.github.com/mlhaufe/322ff76955ce1426e850df87e62432c9
